### PR TITLE
feat(DEV-1228): basic component requirements

### PR DESCRIPTION
**Pull request by milldr**
## what
Testing PR in template

## why
- This should trigger organization required checks

## references
- DEV-1228

### CODE REVIEW

**Comment by milldr** [comment]
after these workflows are completed, make sure to pin to commit SHA, not branch:

CleanShot 2024-01-12 at 15 19 56

File: .gitattributes [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# Note, this can be removed later. Currently keeps PR's easier to review
+docs linguist-documentation=true
+**/*.md linguist-documentation=true
```
Comment (osterman) [inline .gitattributes:3]: Can multiple linguist tags be used so that this also tagged as generated

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+**/.Trash/
+**/$RECYCLE.BIN/
+
+# deps
+Brewfile.lock.json
+
+# custom docker files
+Dockerfile.mine
+Dockerfile.custom
+
```
Comment (osterman) [inline .gitignore:64]:
```suggestion
```

File: .gitattributes (hunk 1 above)
Comment (osterman) [reply .gitattributes:2]: Looks like it's happening automatically.

**Pull request merged**